index.html

```diff
@@ -281,11 +281,11 @@ <h3><b>Medical Education</b></h3>
                                         <h3><b>Big-Data in Healthcare</b></h3>
                                         <p>Microsoft Azure Gift Card</p>
                                     </li>
-									                  <li>
+								  <li>
```

Comment:
I'm going to merge this, but please fix your editors to use soft tabs with 4 spaces. Use a formatter like Emmett to ensure proper formatting.

Reply:
My editor already does use 4 space soft tabs, but something weird keeps happening when I upload to Github that throws the spacing way off. I'm looking at how to fix it.